tests/testthat/test-plot.easycormatrix.R

```diff
@@ -1,5 +1,7 @@
 test_that("`plot.see_easycormatrix()` works", {
   library(correlation)
+  library(see)
+
```

Comment:
packages should be included with if-require() or if-requireNamespace() - why do we need see here?